release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -462,11 +472,22 @@ Note: You should not have to build the reference documentation for every release
 
 Update the `<MINOR_VERSION>` in `content/en/docs/reference/kubernetes-api/api-index.md` for the future release.
 
-   ![Update API index](pics/update-api-index.png)
+```diff
+---
+- title: v1.19
++ title: v1.20
+weight: 50
+---
+
+- [Kubernetes API v1.19](/docs/reference/generated/kubernetes-api/v1.19/)
++ [Kubernetes API v1.19](/docs/reference/generated/kubernetes-api/v1.20/)
```

Comment:
nit: If you are changing this (not sure if needed as it is just an example?), link text should be updated to `v1.20`.

Reply:
Updated! thank you